### ISSUE TITLE
GPLv3 license

### ISSUE THREAD

**Issue by david-l-riley** [DapperDox/dapperdox#98]
We would really love to use DapperDox in our product once OpenAPI 3 support is present, but the GPLv3 license presents some problems for a commercial Go product for which we cannot disclose source.

As I understand it, because Go programs are compiled from source and linked in, at the very least an LGPL license would be required to use it, but even then that usually refers to libraries built separately and statically linked later. I'm not sure how it washes out with Go packages, which are built from source at build time with the rest of the program.

Do you have any plans to move to a more permissive license, and/or offer commercial licenses?

**Comment by jondreid0** [comment]
Yes this would be an issue for our use of Dapperdox as well.  I work with a metadata-driven commercial system where customers can extend the API, and would like to enabled Dapperdox generation for their extensions as well as our baseline.  Distributing GPL-licensed software with our system is basically a non-starter.
